### ISSUE TITLE
[Bug] no exit portal in end

### ISSUE THREAD

**Issue by HaraldLP** [quiqueck/BetterEnd#130]
### What happened?

ive playing this world for a while now. i played a lot vanilla but recently applied some mods, one of them better end. ive been in the end already and resetted the end via MCA Selector and even deleted the end in the world folder. the problem: i cant get the exit portal in this world. it simply doesnt exist. yes, ive been to the end, killed the dragon and looted some pre mod end cities.
![image](https://user-images.githubusercontent.com/59770470/208109375-f880eb43-c9b5-44f5-8c0e-cd05cd9bc209.png)

### BetterEnd

1.1.1

### BCLib

1.4.7

### Fabric API

0.58.0

### Fabric Loader

0.14.9

### Minecraft

1.18.2

**Comment by TheWyzer** [comment]
I am having the same issue as well as NOT having an Ender Dragon appear, so I cannot vanilla wise leave the end.
BetterEnd  1.1.1
BCLib   1.4.7
Fabric API  0.67.0
Fabric Loader  0.14.11
Minecraft 1.18.2

**Comment by AnxiousWizardry** [comment]
So this doesn't fix the code or anything, but if you're missing an end portal, try to build the basic one and do the ritual to bring back the dragon and it should make it so there's an end portal, though you will need to fight the dragon again. This worked for 1.19.2, but I imagine it should still work for 1.18.2

**Comment by TheWyzer** [comment]
Thank you for the support, was beginning to think this was just a dead issue.

Do you have a link to building the End EXIT Portal by chance?  I have searched a bit, but only find the End Portal from Overworld to End builds, NOT End to Overworld builds.

Sad thing is this Mod seems to stop the vanilla end portal (Overworld to End) from functioning, nor any to spawn with world, as I have tried a few different world and no vanilla portal is placed.

**Comment by TheWyzer** [comment]
No Joy.  I even summoned the Ender Dragon, and killed it, and nothing was created or done.  I built the End to Overworld "dragon alter" portal, and did the summoning ritual, and nothing would trigger.  It is almost as if the "End" for Minecraft doesn't exist.  I have a mod that allow the End Portal to be crafted and built, and it will not activate either.

**Comment by AnxiousWizardry** [comment]
If you can, try doing it in just creative mode by hand with bedrock instead of having the mod do it (if you're allowing cheats of course). It has to be like the exact vanilla exit portal and the vanilla ritual to summon the dragon, though do it after you've deleted the end folder completely so it makes a new folder. I hope we can figure this out for you!

The reason it's happening is most liked the end had already been accessed before the mod was installed (like killing the dragon and going further in) as that's what happened with the server I'm on.

**Comment by TheWyzer** [comment]
I will give it a shot.  As a note, I had this mod in my pack from the start.

I just need to confirm the actual build of the End exit portal/altar.

**Comment by TheWyzer** [comment]
OK, so created an vanilla instance, went to end, killed dragon and copied the End Exit Portal.  I deleted the BetterEnd config folder and the nbt file in the save.  I built the exit at the 0,0 location at the height of the highest blocks, placed the torches and all.  I placed the end crystals to summon the dragon, however the crystals do not "ignite" on placement (no fire under them like in vanilla).  I even tried summoning the dragon via command.  Killed it and nothing, not even the end credits for beating it.  I should also note there is not a single "floating" portal between bedrock pieces in this end.  

I copied this instance, removed the BetterEnd mod and created a new world, cheated to end portal and had no issue with the dragon spawning, killed it, exited the End, came back and summoned the dragon again.
